packages/react-native-gradle-plugin/build.gradle.kts

```diff
@@ -50,6 +52,19 @@ java {
     targetCompatibility = JavaVersion.VERSION_11
```

Comment:
Please change those two to `JavaVersion.VERSION_8` otherwise the build won't even start on JDK 8

Reply:
as far as I know. users of old releases will use older versions of `react-native-gradle-plugin`. let me change it to `JavaVersion.VERSION_1_8`

Reply:
@cortinico with `JavaVersion.VERSION_1_8` i am getting this error
```
Build file '/Users/apple/Data/react-native/ReactAndroid/build.gradle' line: 9

* What went wrong:
An exception occurred applying plugin request [id: 'com.android.library']
> Failed to apply plugin 'com.android.internal.library'.
   > Android Gradle plugin requires Java 11 to run. You are currently using Java 1.8.
     You can try some of the following options:
       - changing the IDE settings.
       - changing the JAVA_HOME environment variable.
       - changing `org.gradle.java.home` in `gradle.properties`.
```

Reply:
Are you building with JDK 11 right?

Reply:
@cortinico I am getting this error for java 1.8. with JDK 11 it is working well. should and replace`JavaVersion.VERSION_11` with`JavaVersion.VERSION_1_8`, I think, we should not replace it because there is no benefit according to me what do you say?